From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Wed, 5 Feb 2014 12:16:12 +0100
Subject: [PATCH] resolve existing LD_LIBRARY_PATH during make, not during
 configure

Otherwise, calling "make install" with fakeroot may not work correctly.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 863d7929507b..661cd3dcb76c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -926,7 +926,7 @@ if test $enable_shared = "yes"; then
     Linux*|GNU*|NetBSD*|FreeBSD*|DragonFly*|OpenBSD*)
 	  LDLIBRARY='libpython$(VERSION).so'
 	  BLDLIBRARY='-L. -lpython$(VERSION)'
-	  RUNSHARED=LD_LIBRARY_PATH=`pwd`${LD_LIBRARY_PATH:+:${LD_LIBRARY_PATH}}
+	  RUNSHARED=LD_LIBRARY_PATH=`pwd`:'${LD_LIBRARY_PATH:+:${LD_LIBRARY_PATH}}'
 	  case $ac_sys_system in
 	      FreeBSD*)
 		SOVERSION=`echo $SOVERSION|cut -d "." -f 1`
